xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/adapter/BaseAccountInformationService.java

```diff
@@ -63,6 +63,7 @@ public BaseAccountInformationService(String baseUri,
 
     protected <T> Response<ConsentCreationResponse> createConsent(RequestHeaders requestHeaders, Consents body, Class<T> klass, Function<T, ConsentCreationResponse> mapper) {
         Map<String, String> headersMap = populatePostHeaders(requestHeaders.toMap());
+        headersMap = addPsuIdHeader(headersMap);
```

Comment:
Is this really needed to be default? I have tested couple of banks with their sandbox environments and none of them requires PSU-Id for AIS

Reply:
Hi, Zeljko! 
We received a bug report, that some production environments require this header to not to be missing. That's why we have to add it somehow.
Also, I was able to reproduce this issue with Fiducia sandbox environment (but I didn't dive deep with their sandbox this time).

Reply:
Thanks for answer, I hope so that other environments which are working without this header will not return errors because of header with empty value.